plugins/modules/zos_copy.py

```diff
@@ -3053,7 +3057,7 @@ def run_module(module, arg_def):
 def main():
     module = AnsibleModule(
         argument_spec=dict(
-            src=dict(type='path'),
+            src=dict(type='str'),
```

Comment:
At first sight this change might look dangerous, if we change the src type to `str` from `path` then validations performed by BetterArgParser will not work? No, after checking the code `argument_spec` is not validated by BetterArgParser but rather `arg_def` which will keep the `type='data_set_or_path'` definition, so validation on `src` will continue to be performed by BetterArgParser, I think we don't have a test case for it but neither we need it.

From what I see this change looks ok.

Reply:
In new modules are work properly with str as type, because ansible is expecting primitives data types